CODEOWNERS

```diff
@@ -5,9 +5,37 @@
 # * last matching line wins (so order from most general to most specific)
 # * multiple elements on the same line means "OR"
 
-# default: unless more specific rule is found, these will be the reviewers assigned; should be a broad group
-* @fedimint/Reviewers
+# default: unless more specific rule is found, these will be the reviewers assigned
+# this will be a chore for people here, but guarantees that *someone* gets to look
+# at every PR (they can unassign and unsubscribe themselves after taking an initial look)
+* @fedimint/firehose
```

Comment:
```rust
for path in each_file_changed_in_a_commit {
  if let Some(group) = find_last_rule_in_codowner_that_matches(path) {
    self.assign_review_request(group);
  }
}
```

Since that rule is the first one (last one to match), and matches everything, any file that didn't find a better match will assign review request to `firehose` group.